gui/aboutdialog.cpp

```diff
@@ -18,22 +18,25 @@
 
 #include "aboutdialog.h"
 
+#include "ui_about.h"
+
 #include <QDialogButtonBox>
 #include <QLabel>
 
 AboutDialog::AboutDialog(const QString &version, const QString &extraVersion, QWidget *parent)
     : QDialog(parent)
+    , mUI(new Ui::About)
```

Comment:
`mUI` will not be deallocated right?

Reply:
This is a pattern what was used in existing classes - so if we leak those we already did several others before. And don't those Qt classes deallocate themselves?

Since this is an existing pattern I think we could merge this. I will check for memory leaks though and add a follow-up PR which fixes all instances if there are any.

Reply:
Even without these changes there are leaks reported for `cppcheck-gui`:
```
==12148==    still reachable: 410,228 bytes in 6,593 blocks
```

I will check those tomorrow (if they can be fixed) and prepare an additional PR fixing them. I will also file tickets in case there are some since we currently report nothing.

Reply:
Ah - I just checked the existing ones and they do delete those objects. Will fix immediately.

Reply:
Done. Will file tickets for the missed memory leaks tomorrow.

Reply:
Filed https://trac.cppcheck.net/ticket/10884.